public/resources/default-files/html.txt

```diff
@@ -1,15 +1,13 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Made with Thimble</title>
-</head>
-<body>
-
-  <h1>Welcome to Thimble</h1>
-
-  <p>Make something <strong>amazing</strong> with the web!</p>
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <title>Made with Thimble</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
```

Comment:
Oh weird, you're right - I'm not sure why it doesn't appear in the original. To me it looks like we'll just be adding the style.css file one, on line 6. Am I totally misreading something?